DNN Platform/Website/Install/InstallWizard.aspx

```diff
@@ -666,7 +675,8 @@
                             databaseSecurity: "",
                             databaseUsername: "",
                             databasePassword: "",
-                            databaseRunAsOwner: null
+                            databaseRunAsOwner: null,
+                            dnnImprovementProgram: $('#<%= chkImprovementProgram.ClientID %>')[0].is(":checked") ? "Y" : "N"
```

Comment:
This was fixed after the code review by a direct commit into the release branch. 
https://github.com/dnnsoftware/Dnn.Platform/commit/b0f64827d07286176d4e2bfc4c9d8a6c06fa1c6a